gradio/layouts.py

```diff
@@ -296,3 +296,42 @@ def style(
 class Form(BlockContext):
     def get_config(self):
         return {"type": "form", **super().get_config()}
+
+
+@document()
+class Accordion(BlockContext):
+    """
+    Accordion is a layout element which can be toggled to show/hide the contained content.
+    Example:
+        with gradio.Accordion("See Details"):
+            gr.Markdown("lorem ipsum")
+    """
+
+    def __init__(self, label, *, open: bool = True, **kwargs):
+        """
+        Parameters:
+            label: name of accordion section.
+            open: An optional string that is assigned as the id of this component in the HTML DOM. Can be used for targeting CSS styles.
```

Comment:
* Fix docstring for `open`

Reply:
`open` is a Python keyword so syntax highlighters annoyingly highlight it. WDYT about naming this `closed` instead?